src/Viewer.tsx

```diff
@@ -203,9 +200,8 @@ function Viewer(): ReactElement {
    */
   const [frameInput, setFrameInput] = useState(0);
   const [findTrackInput, setFindTrackInput] = useState("");
-  // Prevent jarring jumps in the hover tooltip by using the last non-null value
-  const [lastHoveredId, setLastHoveredId] = useState<number | null>(null);
-  const [showHoveredId, setShowHoveredId] = useState(false);
+  const [lastValidHoveredId, setLastValidHoveredId] = useState<number>(-1);
+  const [showObjectHoverInfo, setShowObjectHoverInfo] = useState(false);
```

Comment:
`-1` is okay here as a placeholder value, because `showObjectHoverInfo` is only set to true when `lastValidHoveredId` can be set to a valid ID number.

Reply:
since you called this out, it's definitely odd to call it a "valid" hovered id when -1 is allowed!